cmd/openshift-install/testdata/agent/image/assets/default_assets.txt

```diff
@@ -40,4 +40,4 @@ apiVersion: v1alpha1
 metadata:
   name: ostest
   namespace: cluster0
-rendezvousIP: 192.168.111.20
\ No newline at end of file
+rendezvousIP: 192.168.111.20
```

Comment:
small typo?

Reply:
I actually didn't make that change so I assume its the editor that removed the newline but I think its correct as these files should not have a newline at the end of file